src/simulation.md

```diff
@@ -307,27 +306,34 @@ Analogous to the `door_supervisor`, a `lift_supervisor` [node](https://github.co
 
-To replicate this delivery behavior in simulation, the `TeleportDispenser` and `TeleportIngestor` [plugins](https://github.com/osrf/rmf_demos/tree/master/rmf_gazebo_plugins/src) are designed. These plugins are loaded into [3D models](https://github.com/osrf/rmf_demos/tree/master/rmf_demo_assets/models) the same names.
-To setup a payload loading station:
-* Assign a `workcell_name` attribute to the waypoint (see figure below)
-* Add a `TeleportDispenser` model beside the waypoint with `name` matching the `workcell_name`
+To replicate the loading and unloading processes in simulation, the `TeleportDispenser` and `TeleportIngestor` [plugins](https://github.com/osrf/rmf_demos/tree/master/rmf_gazebo_plugins/src) are designed. These plugins are attached to the `TeleportDispenser` and `TeleportIngestor` [3D models](https://github.com/osrf/rmf_demos/tree/master/rmf_demo_assets/models) respectively.
+To setup a payload loading station in simulation:
+* Add a `TeleportDispenser` model beside the pickup waypoint and assign it a
+  unique `name`.
 * Add the payload model beside the `TeleportDispenser` model (Coke can in image below)
 
-To setup a payload unloading station:
-* Assign a `workcell_name` attribute to the waypoint (see figure below)
-* Add a `TeleportIngestor` model beside the waypoint with `name` matching the `workcell_name`
+To setup a payload unloading station in simulation:
+* Add a `TeleportIngestor` model beside the dropoff waypoint and assign it a
+  unique `name`.
 
-When a `DispenserRequest` message is published with `target_guid` matching the name of the `TeleportDispenser` model, the plugin will teleport the payload onto the nearest robot model. Conversely, when the `target_guid` matches the name of the `TeleportIngestor` model, the `TeleportIngestor` plugin will teleport the payload on the robot to its location in the world. The combinations of these plugins allow for delivery requests to be simulation. In the future, this mechanism will be replaced by actual workcells or robot arms but the underlying message exchanges will remain the same.
+When a `DispenserRequest` message is published with `target_guid` matching the
```

Comment:
I see the term repeated but is `target_guid` intentionally leaving off the trailing `e`?

Reply:
`target_guid` is a field in the DispenserRequest message definition
https://github.com/osrf/rmf_core/blob/66b34c5968392f0e9515d69edc9489570fbeae23/rmf_dispenser_msgs/msg/DispenserRequest.msg#L7

Reply:
thx!